client/commonFramework/create-editor.js

```diff
@@ -37,120 +36,6 @@ window.common = (function(global) {
     }
   );
 
-  var clipboard = new Clipboard('.copy-btn', {
-    text: function(trigger) {
```

Comment:
Do you mean something like creating a new `editor-clipboard.js` file in `client/commonFramework/` and moving `var clipboard = new Clipboard(...` code block there?